docs/UserGuide.adoc

```diff
@@ -298,6 +307,9 @@ The `redo` command fails as there are no `undo` commands executed previously.
 Clears all entries from the application. +
 Format: `clear`
 
+The rooms are maintained - only all their current and future bookings are
+cleared. I.e. all maintenance requests and faults will still be recorded.
```

Comment:
Any particular reason why clear should maintain fault statuses? I think clear should reset everything (i.e. room list becomes just like `new UniqueRoomList()`)

Reply:
Hmm, the rational was that clear deletes bookings only. In the context of a hotel operation, it would be more likely that the user will want to

clear bookings only and maintain room faults, as faults have to be rectified

than

clear all bookings and room faults

Reply:
Can easily solve this using flags that call different versions of the clear command.